src/avdictionary.jl

```diff
@@ -29,6 +29,7 @@ end
 Base.empty!(d::AVDict) = libffmpeg.av_dict_free(d.ref_ptr_dict)
 
 Base.convert(::Type{Ref{Ptr{AVDictionary}}}, d::AVDict) = d.ref_ptr_dict
```

Comment:
If this method is not used anywhere, it's better to remove it to avoid further confusion.